### PR TITLE
Enforce Prettier formatting on markdown sources.

### PR DESCRIPTION
**Pull request by olafurpg**
Previously, the CI did not check the formatting of markdown files. Now,
we require that users run Prettier on the markdown sources with the
following command
```
npx prettier@2.0.4 --write docs
```

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ title: Changelog
 
 [Full Changelog](https://github.com/scalameta/scalafmt/compare/v2.4.1...v2.4.2)
 
+Hello
+worl
```
Comment (kitbellew) [inline docs/CHANGELOG.md]: 1. typo
2. should it be here?

Comment (kitbellew) [reply docs/CHANGELOG.md]: never mind, disappeared.

Comment (olafurpg) [reply docs/CHANGELOG.md]: I used this to test that the CI would fail

**Comment by kitbellew** [comment]
> I don't mind using `npm` if that's preferred 🤷‍♂

not by me. i have no preference at all.

p.s. looks like different yarn. people complain of name conflicts.

**Pull request merged**